community/commercial-support.tt

```diff
@@ -15,10 +15,273 @@
+        <ul><li>Ohio, USA</li></ul>
+        Black River Software offers custom software development, software architecture
+consulting, and build and deployment engineering services.
+      </a>
+    </li>
+
+    <li>
+      <a href="https://determinate.systems/">
+        <div>
+          <img alt="Determinate Systems" src="/community/commercial-support-logos/determinate-systems.svg" />
+        </div>
+        <h2>Determinate Systems</h2>
+        <ul><li>Remote</li><li>Massachusetts, USA</li></ul>
+        Confidently build and deploy to the cloud, stadium, or stock exchange.
+Expert help with the Nix ecosystem from Graham Christensen and
+collaborators.
+      </a>
+    </li>
+
+    <li>
+      <a href="https://www.fivebinaries.com/">
+        <div>
+          <img alt="five binaries" src="/community/commercial-support-logos/5inaries.png" />
+        </div>
+        <h2>five binaries</h2>
+        <ul><li>Tallinn, Estonia</li><li>Prague, Czech Republic</li></ul>
+        Five Binaries are an infrastructure development company focused on creating
+customized highly reliable solutions for blockchain-powered industries.
+      </a>
+    </li>
+
+    <li>
+      <a href="https://nixcademy.com">
+        <div>
+          <img alt="Nixcademy" src="/community/commercial-support-logos/nixcademy.svg" />
+        </div>
+        <h2>Nixcademy</h2>
+        <ul><li>Remote</li><li>Germany</li></ul>
+        Elevate your organization's expertise with Nix and NixOS through our comprehensive corporate training classes, personalized mentoring, and expert consulting services.
+Empower your team with the latest knowledge and best practices.
+      </a>
+    </li>
+
+    <li>
+      <a href="https://obsidian.systems">
+        <div>
+          <img alt="Obsidian Systems" src="/community/commercial-support-logos/obsidian-systems.svg" />
+        </div>
+        <h2>Obsidian Systems</h2>
+        <ul><li>New York, USA</li></ul>
+        Software consultancy specializing in Haskell, Nix, curiosity, and innovation.
+      </a>
+    </li>
+
+    <li>
+      <a href="https://platonic.systems/">
+        <div>
+          <img alt="Platonic.Systems" src="/community/commercial-support-logos/platonic-systems.svg" />
+        </div>
+        <h2>Platonic.Systems</h2>
+        <ul><li>Remote</li></ul>
+        Solving your toughest software problems
+Trying to build an innovative software solution? Is it behind schedule?
+Over budget? Not working properly? Growing with bugs? We can help.
+      </a>
+    </li>
+
+    <li>
+      <a href="https://nix.how">
+        <div>
+          <img alt="Nix.How" src="/community/commercial-support-logos/nix-how.png" />
+        </div>
+        <h2>Nix.How</h2>
+        <ul><li>Remote</li><li>United Kingdom</li></ul>
+        Nix experts specialising in converting legacy infrastructure to a Nix/NixOS
+methodology and workflow
+      </a>
+    </li>
+
+    <li>
+      <a href="https://nixos.mayflower.consulting">
```

Comment:
This does not look correct, fixed in https://github.com/NixOS/nixos-homepage/pull/1140